docs/external_system.md

```diff
@@ -46,13 +46,13 @@ use externally-accessible API endpoints exposed by CSM.
    $ . ${SAT_VENV_PATH}/bin/activate
    ```
 
-1. Clone the SAT source code.
+1. (`venv`) Clone the SAT source code.
 
    To use SAT version 3.21, this example clones the `release/3.21` branch of
    `Cray-HPE/sat`.
 
    ```screen
-   (venv) $ git clone --branch=release/3.21 https://github.com/Cray-HPE/sat.git
+   $ git clone --branch=release/3.21 https://github.com/Cray-HPE/sat.git
```

Comment:
I won't comment on every one of these, but we should remove the `$ ` from the beginning of each of the command examples in this file. We can also stop using `screen` and use `bash` or `sh` instead now that our commands should follow bash or sh syntax since they don't include the command prompt. Something like this:


````
   ```sh
   git clone --branch=release/3.21 https://github.com/Cray-HPE/sat.git
   ```
````

Reply:
Updated in next commit.